pkg/ccl/changefeedccl/topic.go

```diff
@@ -185,7 +185,7 @@ func (tn *TopicNamer) makeName(s changefeedbase.Target, td TopicDescriptor) (str
 		return tn.nameFromComponents(s.StatementTimeName, s.FamilyName), nil
 	case jobspb.ChangefeedTargetSpecification_EACH_FAMILY:
 		if td == nil {
-			return tn.nameFromComponents(s.StatementTimeName, familyPlaceholder), nil
+			return tn.nameFromComponents(s.StatementTimeName) + string(tn.join) + familyPlaceholder, nil
```

Comment:
i can see how this would help the printed name, but wouldnt this also cause the phony topic created in topic to have an invalid name, and result in an error?